src/extension.ts

```diff
@@ -259,47 +256,13 @@ export async function activate(context: ExtensionContext) {
 		return disposable;
 	}
 
-	function getDefaultVenvPath() {
-		if (os.platform() === "win32") {
-			return Uri.joinPath(context.globalStorageUri, ".env", "Scripts", "python.exe").fsPath;
-		}
-		return Uri.joinPath(context.globalStorageUri, ".env", "bin", "python").fsPath;
-	}
-
-	async function createVenvIfNotExists() {
-		const pp: string = workspace.getConfiguration('inmanta').pythonPath;
-		if (pp && fs.existsSync(pp)) {
-			return pp;
-		} else {
-			window.showInformationMessage(`No Python3 interpreter found at "${pp}". Falling back to default virtual environment`);
-		}
-		const venvBaseDir = Uri.joinPath(context.globalStorageUri, ".env").fsPath;
-		const venvPath = getDefaultVenvPath();
-
-		if (!fs.existsSync(venvBaseDir)) {
-			log("Creating new virtual environment");
-			const venvProcess = cp.spawnSync("python3", ["-m", "venv", venvBaseDir]);
-			if (venvProcess.status !== 0) {
-				window.showErrorMessage(`Virtual env creation at ${venvBaseDir} failed with code ${venvProcess.status}, ${venvProcess.stderr}`);
-			}
-			log("Ensuring latest pip and wheel");
-			const updateProcess = cp.spawnSync(venvPath, ["-m", "pip", "install", "-U", "pip", "wheel"]);
-			if (updateProcess.status !== 0) {
-				window.showErrorMessage(`Updating pip and wheel in venv ${venvBaseDir} failed with code ${updateProcess.status}, ${updateProcess.stderr}`);
-			}
-			installLanguageServer(venvPath);
-		}
-		await workspace.getConfiguration("inmanta").update("pythonPath", venvPath, true);
-		return venvPath;
-	}
-
 	function registerExportCommand() {
 		const commandId = 'inmanta.exportToServer';
 
 		const commandHandler = (openedFileObj: object) => {
 			const pathOpenedFile: string = String(openedFileObj);
 			const cwdCommand: string = path.dirname(Uri.parse(pathOpenedFile).fsPath);
-			const pythonPath: string = workspace.getConfiguration('inmanta').pythonPath;
+			const pythonPath : string = pythonExtentionApi.pythonPath;
```

Comment:
This is called `pp` everywhere. Maybe keep it consistent.
And do you need an actual variable to hold this value?
Or can you just use `pythonExtentionApi.pythonPath` everywhere?

Reply:
indeed, i'll use pythonExtentionApi.pythonPath everywhere :)